bbcdll/bbc_tcp_server.py

```
@@ -619,13 +668,23 @@ def get_ui_status():
                 'battle_running': False
             }
         try:
-            # 获取topLabel文本
-            top_label_text = ''
-            if hasattr(page, 'topLabel'):
-                try:
-                    top_label_text = page.topLabel.cget('text')
-                except:
-                    pass
+            # 线程安全：cget() 严格意义上非线程安全，但实践中较少崩溃
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)